### PR TITLE
Feature: Add method for creating text/html response

### PR DESCRIPTION
**Pull request by jczuchnowski**
I'm doing some experiments with serving static content and though this could be useful  in addition to `jsonString` and `text` methods on `Response`.

### CODE REVIEW

**Comment by girdharshubham** [comment]
Hi @jczuchnowski 
A better api design would be using Zstream instead. In it, the method would take in the path of the HTML file and use Zstream to read it.

**Comment by jczuchnowski** [comment]
@girdharshubham your suggestion makes the use case very narrow. This method could be easily used also to serve some arbitrary html assembled in the application.

But maybe it would make sense to have a couple of overloaded methods that can take a String or a Path? And maybe a ZStream version too?

**Comment by girdharshubham** [comment]
@jczuchnowski  Yes, that would be great!

File: zio-http/src/main/scala/zhttp/http/Response.scala [hunk 1]
```diff
@@ -92,6 +92,15 @@ object Response {
     }
   }
 
+  /**
+   * Creates a response with content-type set to text/html
+   */
+  def html(data: String): UResponse =
+    Response(
+      data = HttpData.fromChunk(Chunk.fromArray(data.getBytes(HTTP_CHARSET))),
```
Comment (tusharmath) [inline zio-http/src/main/scala/zhttp/http/Response.scala]:
```suggestion
      data = HttpData.fromText(data),
```

Can keep `charset` as an argument to `html()` defaulting to `HTTP_CHARSET`.

**Comment by tusharmath** [comment]
One small change @jczuchnowski. Looks good other wise :)

**Review by tusharmath — APPROVED** [review]

**Pull request merged**